src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,27 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ moviesProperties }) => (
+  <div className="movies">
+    {moviesProperties.map(card => (
+      <MovieCard
+        {...card}
+        key={card.imdbId}
+      />
+    ))}
+  </div>
 );
+
+MoviesList.propTypes = {
+  moviesProperties: PropTypes.arrayOf(
```

Comment:
moviesProperties - why do you add properties suffix?
just `movies` would be enugh